src/components/WelcomePage/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./WelcomePage.css";
+
+const Footer = (props) => {
+    return (
+        <footer>
```

Comment:
i would do something like this 
const navitems=["About", "Help xxx",xxx,xxx]
<ul class="footerList">{navitems.map(ele => <li></li>)}</ul>

Reply:
is it more conventional to map it through js rather than just writing them out? i figured since they are static items, i just typed them out.

Reply:
what you have is fine. i feel that later you can refactor this once ur code base get bigger